Specs/spec-main.js

```diff
@@ -136,6 +136,8 @@
 
         window.it = function(description, f, timeout, categories) {
             originalIt(description, function(done) {
+                // Clear the RequestScheduler before for each test in case requests are still active from previous tests
+                Cesium.RequestScheduler.clearForSpecs();
```

Comment:
Right, it should only matter for tests that are testing the `RequestScheduler` itself.